setup/terraform/idp-base.tf

```diff
@@ -50,6 +50,7 @@ resource "humanitec_resource_definition_criteria" "k8s_namespace" {
 # Configure DNS for localhost
 
 resource "humanitec_resource_definition" "dns_localhost" {
+  depends_on = [helm_release.operator]
```

Comment:
We need this, not to remove the operator before resources are de-provisioned (the definition is not removed until active resources exist).